extras/indexingExample/src/main/java/RyaDirectExample.java

```diff
@@ -156,8 +157,8 @@ private static Configuration getConf() {
 		conf.set(ConfigUtils.USE_GEO, "true");
 		conf.set(ConfigUtils.USE_FREETEXT, "true");
 		conf.set(ConfigUtils.USE_TEMPORAL, "true");
-		conf.set(RdfCloudTripleStoreConfiguration.CONF_TBL_PREFIX,
-				RYA_TABLE_PREFIX);
+		conf.set(PrecomputedJoinIndexerConfig.PCJ_STORAGE_TYPE, PrecomputedJoinStorageType.ACCUMULO.name());
+		conf.set(RdfCloudTripleStoreConfiguration.CONF_TBL_PREFIX, RYA_TABLE_PREFIX);
```

Comment:
Done with review. I'm cool with it going in after you address these comments. It's all nits.